listenbrainz/webserver/views/api.py

```diff
@@ -256,10 +256,12 @@ def get_similar_users(user_name):
     Get list of users who have similar music tastes (based on their listen history)
     for a given user. Returns an array of dicts like these:
 
-    {
-      "user_name": "hwnrwx",
-      "similarity": 0.1938480256
-    }
+    .. code-block:: json
+
+        {
+        "user_name": "hwnrwx",
+        "similarity": 0.1938480256
+        }
```

Comment:
```suggestion
        {
            "user_name": "hwnrwx",
            "similarity": 0.1938480256
        }
```
Please update the indentation for all other jsos too.